docs/getting-started/installing-and-running.mdx

```diff
@@ -0,0 +1,144 @@
+
+For an ARM based back Mac for example we'd grab the `conduit_0.10.0_Darwin_arm64.tar.gz` archive.
+
+### unzip the archive
+
+```shell
+tar xzvf conduit_0.10.0_Darwin_arm64.tar.gz
+```
+
+### Run
+
+In the directory in which you expanded the archive. Execute the Conduit binary:
+
+```shell
+./conduit
+```
+
+```shell
+             ....
+         .::::::::::.
+       .:::::‘‘‘‘:::::.
+      .::::        ::::.
+ .::::::::          ::::::::.
+ `::::::::          ::::::::‘
+      `::::        ::::‘
+       `:::::....:::::‘
+         `::::::::::‘        Conduit v0.10.0 linux/amd64
+             ‘‘‘‘
+2024-02-20T21:37:45+00:00 INF All 0 tables opened in 0s component=badger.DB
+2024-02-20T21:37:45+00:00 INF Discard stats nextEmptySlot: 0 component=badger.DB
+2024-02-20T21:37:45+00:00 INF Set nextTxnTs to 0 component=badger.DB
+2024-02-20T21:37:45+00:00 INF builtin plugins initialized component=builtin.Registry count=6
+2024-02-20T21:37:45+00:00 WRN could not read plugin directory error="open /app/connectors: no such file or directory" component=standalone.Registry
+2024-02-20T21:37:45+00:00 INF standalone plugins initialized component=standalone.Registry count=0 plugin_path=/app/connectors
+2024-02-20T21:37:45+00:00 INF processors initialized component=processor.Service count=0
+2024-02-20T21:37:45+00:00 INF connectors initialized component=connector.Service count=0
+2024-02-20T21:37:45+00:00 INF pipelines initialized component=pipeline.Service count=0
+2024-02-20T21:37:45+00:00 INF pipeline configs provisioned component=provisioning.Service created=[] deleted=[] pipelines_path=./pipelines
+2024-02-20T21:37:45+00:00 INF grpc server started address=[::]:8084
+2024-02-20T21:37:45+00:00 INF http server started address=[::]:8080
+2024-02-20T21:37:45+00:00 INF
+2024-02-20T21:37:45+00:00 INF click here to navigate to Conduit UI: http://localhost:8080/ui
+2024-02-20T21:37:45+00:00 INF click here to navigate to explore the HTTP API: http://localhost:8080/openapi
+2024-02-20T21:37:45+00:00 INF
+```
+
+You should now be able to interact with the Conduit UI and HTTP API on port 8080.
+  </TabItem>
+  <TabItem value="source" label="Building from source">
+Before you can build Conduit from source, you need to have Go 1.21 or higher installed. The Go website has [comprehensive instructions](https://go.dev/doc/install) on how to get the language installed on your machine. Once installed, you can follow the rest of the instructions.
+
+1. Start by downloading the source code from the latest stable release on the Conduit [Releases Page](https://github.com/ConduitIO/conduit/releases/latest). Alternatively, you can run this command to automatically download the latest stable source to your current directory:
+
+```shell
+$ TAG=$(curl -s https://api.github.com/repos/ConduitIO/conduit/releases/latest | grep "tag_name\": \"v[0-9.]*" | awk -F \" '/tag_name\": \"*\"/{print $(NF-1)}') && curl -o conduit.tgz -L https://github.com/ConduitIO/conduit/tarball/$TAG
```

Comment:
I propose to hardcode the tag here and update it whenever we release (we already have other places in this doc that we need to update, so I don't see much harm in that). Additionally, I would use the URL `https://github.com/ConduitIO/conduit/archive/refs/tags/v0.10.0.tar.gz`, since that tarball contains a folder named `conduit-0.10.0` which then makes it simpler to describe how to `cd` into that folder, as the path won't contain a git SHA in the name.

```suggestion
$ TAG=v0.10.0 curl -o conduit.tar.gz -L https://github.com/ConduitIO/conduit/archive/refs/tags/$TAG.tar.gz
```